src/core/loss_detection.c

```diff
@@ -766,6 +766,10 @@ QuicLossDetectionRetransmitFrames(
         QuicSentPacketPoolReturnPacketMetadata(&Connection->Worker->SentPacketPool, Packet);
     }
 
+    if (NewDataQueued) {
+        QuicPerfCounterAdd(PacketsRetransmitted, 1);
```

Comment:
I thought we already had a place where we calculated retransmitted bytes for the connection... If so, we should use that to increment this probably.

Reply:
There's a per-connection counter for retransmittable packets, but that's just packets eligible for retransmission.